### PR TITLE
Scripts/add truffle fixtures

### PR DESCRIPTION
**Pull request by cds-amal**
This PR adds `solidity-magic-square` to the truffle project fixtures as a few helper scripts to help the developer experience.

New scripts:
  * `yarn seed:fixtures` will invoke `truffle migrate --reset` an all fixture projects. ganache-cli is expected to be running
* `yarn start:magicsquare` starts graphiql playground sourcing from magicsquare
* `yarn start:magicsquare:debug` same as above, but set pouch-db's debug environment flag

### CODE REVIEW

File: packages/truffle-db/scripts/deploy-truffle-fixtures.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+function bummer {
+  printf "Well, this is unexpected :/\n"
+  printf "Try running with set -ex to debug...\n\n\n"
+}
+trap bummer ERR
+
+projects="test/truffle-projects"
+for proj in `ls $projects`; do
+  echo "truffle migrate --reset $proj"
```
Comment (gnidan) [inline packages/truffle-db/scripts/deploy-truffle-fixtures.sh]: also needs `--compile-all`

File: packages/truffle-db/test/truffle-projects/solidity-magic-square/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+solidity-magic-square
+=====================
```
Comment (gnidan) [inline packages/truffle-db/test/truffle-projects/solidity-magic-square/README.md]: I don't think this should get duplicated here. Can this be a submodule perhaps, linking back to the original repo at github.com/trufflesuite/solidity-magic-square?

Comment (gnidan) [reply packages/truffle-db/test/truffle-projects/solidity-magic-square/README.md]: a plain `git clone` might be fine also

Comment (cds-amal) [reply packages/truffle-db/test/truffle-projects/solidity-magic-square/README.md]: Yea, the PR to solidity-magic-square broke it spectacularly. I'm looking at a git-clone option w/ a tag or branch specifically for being a fixture for truffle-db.

**Pull request closed without merging**